plugins/circleci/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx

```diff
@@ -15,20 +15,22 @@
  */
 import React, { FC, useEffect } from 'react';
 import { useParams } from 'react-router-dom';
-import { Content, InfoCard, Progress } from '@backstage/core';
+import { InfoCard, Progress, Link } from '@backstage/core';
 import { BuildWithSteps, BuildStepAction } from '../../api';
-import { Grid, Box, Link, IconButton } from '@material-ui/core';
+import {
+  Grid,
+  Box,
+  IconButton,
+  Breadcrumbs,
+  Typography,
+  Link as MaterialLink,
+} from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
-import { PluginHeader } from '../../components/PluginHeader';
 import { ActionOutput } from './lib/ActionOutput/ActionOutput';
-import { Layout } from '../../components/Layout';
 import LaunchIcon from '@material-ui/icons/Launch';
-import { useSettings } from '../../state/useSettings';
 import { useBuildWithSteps } from '../../state/useBuildWithSteps';
-import { AppStateProvider } from '../../state';
-import { Settings } from '../../components/Settings';
 
-const IconLink = IconButton as typeof Link;
+const IconLink = (IconButton as any) as typeof MaterialLink;
```

Comment:
Why is ```any``` needed here?

Reply:
Typescript started to yell, dunno why it wasn't an issue before 🤷